airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -301,36 +301,39 @@ def create_custom_object(
     ):
         """
         Creates custom resource definition object in Kubernetes
-
         :param group: api group
         :param version: api version
         :param plural: api plural
         :param body: crd object definition
         :param namespace: kubernetes namespace
         """
         api = client.CustomObjectsApi(self.api_client)
+
         if namespace is None:
             namespace = self.get_namespace()
-        if isinstance(body, str):
-            body_dict = _load_body_to_dict(body)
-        else:
-            body_dict = body
-        try:
-            api.delete_namespaced_custom_object(
-                group=group,
-                version=version,
-                namespace=namespace,
-                plural=plural,
-                name=body_dict["metadata"]["name"],
-            )
-            self.log.warning("Deleted SparkApplication with the same name.")
-        except client.rest.ApiException:
-            self.log.info("SparkApp %s not found.", body_dict['metadata']['name'])
+
+        body_dict: dict = _load_body_to_dict(body) if isinstance(body, str) else body
+
+        # Attribute "name" is not mandatory if "generateName" is used instead
+        if "name" in body_dict["metadata"].keys():
```

Comment:
```suggestion
        if "name" in body_dict["metadata"]:
```

Reply:
Hi Kamil! Thank you for joining reviewing my PR. I use exact .keys() mention because its more definitely.  Is it bad style?

Reply:
yeah you don't need to, it's commonly accepted to do `if key in dict`

Reply:
Daniel, thank you, I will follow that style. I see someone made a commit - so I do not need to make an additional patch?